src/Windows/Avalonia.Win32/OpenGl/WglDisplay.cs

```diff
@@ -64,20 +67,20 @@ static bool InitializeCore()
                 return false;
 
             wglMakeCurrent(_bootstrapDc, _bootstrapContext);
-            WglCreateContextAttribsArb = Marshal.GetDelegateForFunctionPointer<WglCreateContextAttribsARBDelegate>(
+            s_wglCreateContextAttribsArb = Marshal.GetDelegateForFunctionPointer<WglCreateContextAttribsARBDelegate>(
```

Comment:
@grokys I think we should remove the `s_` prefix for static fields rule, it leads to this kind of naming.

Reply:
Should IDE1006 Be Disabled? #9608

Reply:
I don't understand why this name is weird, I'm probably missing something... Is it because it's a function?

Reply:
@robloo @kekekeks IDE1006 was already configured to be applied only on non-public members. I.e., so we won't have problem with Avalonia property declarations. For the private fields I am not sure. Personally, I don't mind it, but also would agree on removing.